### DISCUSSION TITLE
TFT screen V1.1 show 1/3 of screen in mirror

### DISCUSSION THREAD

**Discussion by DenPrint** [moonglow/FlashForge_Marlin#40] [General]
Hello,

I have installed Marlin on my old Dreamer that i bought 2nd hand.
With the Original FlashForge Firmware the TFT does work fine.

After installing Marlin it does not, only show 1/3 of the screen in mirror.
 see video :  https://youtu.be/ac4yN4e7cRY

**After switching back to the original FlashForge firmware the TFT did work fine again**

I switched the TFT V1.1 to a V5.3 after that it did work ok with Marlin.
So the probleem is in the version of the TFT.

**Comment by moonglow** [comment]
Got it ! I have some ideas about it, i will provide you some test builds soon. Thank you for extended report.

**Reply by moonglow** [reply]
@DenPrint 
Can you check this build, i disable re-config of TFT display controller from Marlin and it stays as it was configured by bootloader. If it fix issue i will allow user to select how to configure TFT display ( it will also speed it up fw loading )

[dreamer_2.0.9.1_07072021_tft.zip](https://github.com/moonglow/FlashForge_Marlin/files/6778277/dreamer_2.0.9.1_07072021_tft.zip)

**Reply by DenPrint** [reply]
I have tested it, the screen is now full screen but still in mirror.
See the pictures
![IMG_0644](https://user-images.githubusercontent.com/86926224/124797860-0be88700-df53-11eb-8fb2-3ee9e335c5df.JPG)
![IMG_0645](https://user-images.githubusercontent.com/86926224/124797903-14d95880-df53-11eb-90be-a8709e3feb6d.JPG)

**Reply by moonglow** [reply]
Ok, that's looks better... but without actual TFT module it is like dice dropping, but i think we finds needed settings. As i can see now screen has local Y-mirrored block. I will also try to find a way how we can readout TFT controller ID and other info.

**Reply by moonglow** [reply]
@DenPrint i do not forget about that problem, but i need some free time to do experiments about controller ID to fix that issue :(

**Reply by DenPrint** [reply]
No problem i have bought a second TFT V5.3, **so take your time** :)
when i need to test something for the V1.1 just lett me know.

Thanks,

**Reply by moonglow** [reply]
@DenPrint Hi ! If that TFT screen is not need to you anymore, maybe you can send it to me ? I want to check that's wrong with it

**Reply by DenPrint** [reply]
Hello,
No problem i dont use it anymore, So yes i can ship it to you.
I send u a private message so u can reply with the shipping adress.
